scipy/stats/_distribution_infrastructure.py

```diff
@@ -3572,7 +3679,12 @@ def _moment_standard_formula(self, order, **kwargs):
                '_entropy': '_entropy_formula',
                '_median': '_median_formula'}
 
+    # These are not desirable overrides for the new infrastructure
+    skip_override = {'norminvgauss': {'_sf', '_isf'}}
```

Comment:
These were just generic algorithms (which could have just as well been put at the `rv_continuous` level) applied to `norminvgauss`. They weren't playing nicely with the way we pass data around, so might as well just avoid them because we have our own (better) generic algorithms. In fact, for performanc, it might be worth adding additional distributions/methods to this dict where there is something similar.